src/libaktualizr/package_manager/ostreemanager.h

```diff
@@ -42,6 +43,9 @@ class OstreeManager : public PackageManagerInterface {
  public:
   OstreeManager(const PackageConfig &pconfig, const BootloaderConfig &bconfig,
                 const std::shared_ptr<INvStorage> &storage, const std::shared_ptr<HttpInterface> &http);
+  OstreeManager(const PackageConfig &pconfig, const BootloaderConfig &bconfig,
```

Comment:
I think there is no in `bconfig` in this ctor?

Reply:
right but we still need to pass bconfig since PackageManagerInterface still needs it

Reply:
It actually doesn't need it https://github.com/foundriesio/aktualizr/blob/e2482e8426e328dac63e8abd7cc55bd68994e44c/include/libaktualizr/packagemanagerinterface.h#L47. The only reason it is part of the PackageManagerInterface ctor signature is a bit "interesting" way of the package manager factory implementation in libaktualizr. So, we can just use `BootloaderConfig()` but it's not big deal...

Reply:
well no, it does need it (dont pass it and the library will not build); a different matter is that is needed for the wrong reasons....but since the objective is to not change the code, why should I have changed it? 

I could do it if you want, happy to spend the time, but seems conflicting requirements with removing the abstract interface.

Reply:
actually since we are talking about it: is the plan to hack aktualizr so we can get away with the less amount of changes or is the plan to do the aktualizr changes properly? I am ok either way, just need to know....because reworking commits do cost time (and money :) )

Reply:
That's fine.
I didn't mean to change it, just "adapt" to it in the new ostree manager ctor, e.g.

```
OstreeManager::OstreeManager(const PackageConfig &pconfig, std::unique_ptr<Bootloader> bootloader,
                             const std::shared_ptr<INvStorage> &storage, const std::shared_ptr<HttpInterface> &http)
    : PackageManagerInterface(pconfig, BootloaderConfig(), storage, http), bootloader_{bootloader}
```